src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.ICompilationTracker.cs

```diff
@@ -39,7 +39,6 @@ private interface ICompilationTracker
             Task<Compilation> GetCompilationAsync(SolutionCompilationState compilationState, CancellationToken cancellationToken);
 
             ICompilationTracker FreezePartialState(CancellationToken cancellationToken);
-            ICompilationTracker FreezePartialStateWithDocument(DocumentState docState, CancellationToken cancellationToken);
```

Comment:
i def like ICompTracker getting simpler.  And i think the semantics of 'Freeze the entire state' are much clearer than 'freeze the state, but ensure we have this docState in it'.